.gitmodules

```diff
@@ -1,5 +1,5 @@
 [submodule "libs"]
 	path = libs
-	url = git://github.com/etlegacy/etlegacy-libs.git
+	url = https://github.com/Bonpournickel/etlegacy-libs.git
```

Comment:
is this mistake?

Reply:
Instead of patching any project in `libs`, you  need to send your patch upstream (in this case, OpenAL). We'll not maintain any forked code downstream.

Reply:
Sorry my mistake.
To make etlegacy-libs openal compatible with gcc-10 openal/Alc/bformatdec.h needs to be patched. It was the only way i figure out to point to it. I cannot use easily openal 32bit on my host. It dragged too much dependencies with it.